src/core/Akka.Remote/RemoteSystemDaemon.cs

```diff
@@ -217,15 +217,18 @@ private void HandleDaemonMsgCreate(DaemonMsgCreate message)
             ActorPath childPath;
             if(ActorPath.TryParse(message.Path, out childPath))
             {
-                IEnumerable<string> subPath = childPath.Elements.Drop(1); //drop the /remote
-                ActorPath path = Path/subPath;
+                IEnumerable<string> subPath = childPath.ElementsWithUid.Drop(1); //drop the /remote
+                ActorPath p = Path/subPath;
+                var s = subPath.Join("/");
+                var i = s.IndexOf("#", StringComparison.Ordinal);
+                var childName = i < 0 ? s : s.Substring(0, i); // extract the name without the UID
```

Comment:
Chops the remotely deployed actor's UID off from the name this actor will use inside the remote daemon's `ChildContainer`, since the UID isn't actually part of the name.